README.md

```diff
@@ -179,29 +192,94 @@ try {
 }
 ```
 
-#### Retrieving a file
+##### Example: Storing multiple files
+
+```js
+/**
+ * Example usage to store multiple files.
+ */
+
+// The hashes of the desired files.
+const hashes = [{
+  algorithm: 'SHA-256',
+  value: '8f434346648f6b96df89dda901c5176b10a6d83961dd3c1ac88b59b2dc327aa4',
+}, {
+  algorithm: 'SHA-256',
+  value: 'ba7816bf8f01cfea414140de5dae2223b00361a396177a9cb410ff61f20015ad',
+}];
+
+// First, check if the files are already in COS.
+try {
+  // This triggers a permission prompt. For example:
+  // example.com wants to check if your browser already has files the site needs,
+  // possibly saved from another site. If found, it will use the files without
+  // changing them.
+  // [Allow this time] [Allow on every visit] [Don't allow]
+  const handles = await navigator.crossOriginStorage.requestFileHandles(hashes);
+  // The files exist in COS.
+  for (const handle of handles) {
+    const fileBlob = await handle.getFile();
+    // Do something with the blob.
+    console.log('Retrieved', fileBlob);
+  }
+  return;
+} catch (err) {
+  // If the files weren't in COS, load them from the network and store them in
+  // COS.
+  if (err.name === 'NotFoundError') {
```

Comment:
I assume this is thrown if at least one file of the "bundle" can't be found?

Reply:
Done via https://github.com/tomayac/cross-origin-storage/pull/30/files/cfd89ea5e9b0c6642759c62d2b80f8b915c5a290#r1929092690.